src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: $border-radius;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__title {
+    margin-top: 40px;
+    color: $primary-color;
+    font-size: $font-size-small;
```

Comment:
The variable `$font-size-small` is used here. Ensure that it is defined in your SCSS files to prevent any compilation issues.